src/iosrv/init.c

```diff
@@ -80,7 +80,13 @@ int			dss_core_depth;
 int			dss_core_nr;
 /** start offset index of the first core for service XS */
 int			dss_core_offset;
-
+/** NUMA node to bind to */
+int			numa_node;
+hwloc_bitmap_t core_allocation_bitmap;
+/** a copy of the NUMA node object in the topology */
+hwloc_obj_t	numa_obj;
```

Comment:
Variable names need to be aligned in C code

Reply:
agreed

Reply:
Fixed